doc/en/conf.py

```diff
@@ -141,10 +141,18 @@
 linkcheck_ignore = [
     "https://blogs.msdn.microsoft.com/bharry/2017/06/28/testing-in-a-cloud-delivery-cadence/",
     "http://pythontesting.net/framework/pytest-introduction/",
-    r"https://github.com/pytest-dev/pytest/issues/\d+",
-    r"https://github.com/pytest-dev/pytest/pull/\d+",
+    r"https://pypi\.org/project/pytest.*",
```

Comment:
This matches any project page starting with `pytest`. Like `pytesty`. Why? I don't think I've seen any problems with this host in linkcheck in the past.